### PR TITLE
FAT-9741 Write karate scenarios for 2 and 3 legged transaction DCB.

### PR DESCRIPTION
**Pull request by pavankumar181**
## Purpose
The Purpose of this pr is to add karate tests for 2 and 3 legged flow in mod-dcb and edge-dcb modules.

## Approach
Added karate tests for 2 and 3 legged flow in mod-dcb and edge-dcb modules.

## Evidence
mod-dcb tests:
fat-9741-mod-dcb

edge-dcb tests:
image

### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

### Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,8 +82,8 @@
     <module>mod-copycat</module>
     <module>mod-bulk-operations</module>
     <module>mod-consortia</module>
-<!--    <module>mod-dcb</module>-->
-<!--    <module>edge-dcb</module>-->
+    <module>mod-dcb</module>
```
Comment (SinghAdes) [inline pom.xml:85]: Not sure if this is okay.  
@gurleenkaurbp :- is mod-dcb avilable in karate env. if not this will cause an issue i think.

Comment (pavankumar181) [reply pom.xml]: Disabled both modules.

**Review by gurleenkaurbp — APPROVED** [review]
